idd/Energy+.idd.in

```diff
@@ -448,19 +448,35 @@ PerformancePrecisionTradeoffs,
       \memo This object enables users to choose certain options that speed up EnergyPlus simulation,
       \memo but may lead to small decreases in accuracy of results.
   A1, \field Use Coil Direct Solutions
-      \note If Yes, an analytical or empirical solution will be used to replace iterations in
+      \note If Yes, an analytical or empirical solution will be used to replace iterations in 
       \note the coil performance calculations.
       \type choice
       \key Yes
       \key No
-      \default No
-  A2; \field Zone Radiant Exchange Algorithm
+  A2, \field Zone Radiant Exchange Algorithm
       \note Determines which algorithm will be used to solve long wave radiant exchange among surfaces within a zone.
       \type choice
       \key ScriptF
       \key CarrollMRT
       \default ScriptF
-
+  A3, \field Override Mode
+      \note The increasing mode number roughly correspond with increased speed. A description of each mode 
+      \note are shown in the documentation. When Advanced is selected the N1 field value is used.
+      \type choice
+      \key Normal
+      \key Mode01
+      \key Mode02
+      \key Mode03
+      \key Mode04
+      \key Mode05
+      \key Advanced
+      \default Normal
+  N1; \field MaxZoneTempDiff
+      \note Only used when Override Mode is set to Advanced
+      \type real
+      \minimum 0.0
+      \maximum 0.0
+      \default 0.0
```

Comment:
@JasonGlazer What should the max/min/default be here? All zero won't work. And units deltaC?

Reply:
Sorry, that was an experiment that I never completed. I would guess a reasonable minimum could be 0.1 and the maximum could be 3 and the default should be 0.3 which is the number in EnergyPlus. And yes, the units are deltaC.